docs/aca/05-aca-dapr-pubsubapi/index.md

```diff
@@ -374,23 +406,40 @@ What we've done In the code above is the following:
+If you don't want to bother with signing up for a SendGrid account to send emails, you can just simulate sending emails by returning always `ok` from the `TaskSaved` method as shown below. In the [next module](/aca/06-aca-dapr-bindingsapi/index.md/#1-create-dapr-sendgrid-output-binding-component-file) we will be using a new type of Dapr components to send emails using SendGrid.
 
+```csharp
+// If you prefer not to deal with setting up a SendGrid account then simply always return OK from the TaskSaved method
+  [Dapr.Topic("dapr-pubsub-servicebus", "tasksavedtopic")]
+  [HttpPost("tasksaved")]
+  public async Task<IActionResult> TaskSaved([FromBody] TaskModel taskModel)
+  {
+      _logger.LogInformation("Started processing message with Task Name '{0}'", taskModel.TaskName);
+      // I decided to simulate a call
+      //var sendGridResponse = await SendEmail(taskModel);
+
+      //if (sendGridResponse.Item1)
+      //{
+          return Ok($"SendGrid response status code: 200");
+      //}
+      //return BadRequest($"Failed to send email, SendGrid response status code: {sendGridResponse.Item1}");
+  }
+```
 ### Use Azure Service Bus as a Service Broker for Dapr Pub/Sub API
-Now we will switch our implementation to use Azure Service Bus as a message broker, Redis worked perfectly for local development and testing but we need to prepare ourselves for the cloud deployment, to do so we need to create Service Bus Namespace, then a Topic.
+Now we will switch our implementation to use Azure Service Bus as a message broker. Redis worked perfectly for local development and testing but we need to prepare ourselves for the cloud deployment. To do so we need to create Service Bus Namespace followed by a Topic. A namespace provides a scoping container for Service Bus resources within your application.
 
 ##### 1. Create Azure Service Bus Namespace and a Topic
-You can do this from Azure Portal or use the below PowerShell command to create the services. I will assume you are using the same PowerShell session from the previous module so variables still hold the right values, you need to change the namespace variable as this one should be unique globally across all Azure subscriptions:
+You can do this from Azure Portal or use the below PowerShell command to create the services. We will assume you are using the same PowerShell session from the previous module so variables still hold the right values. You need to change the namespace variable as this one should be unique globally across all Azure subscriptions:
 
 ```powershell
-$NamespaceName="taskstracker"
+$NamespaceName="[your globally unique namespace goes here. e.g. taskstracker-wk-42 where wk are your initials and 42 is the year you were born]"
```

Comment:
Let's add the sku explicitly, `sku standard` is the default if not passed. Topics only available on the standard tier not on the basic. 

`az servicebus namespace create --resource-group $RESOURCE_GROUP --name $NamespaceName --location $LOCATION --sku Standard`

More details here: https://learn.microsoft.com/en-us/cli/azure/servicebus/namespace?view=azure-cli-latest#az-servicebus-namespace-create-optional-parameters
<img width="310" alt="image" src="https://user-images.githubusercontent.com/3114431/223735567-4e42fefa-01ba-4d40-87d0-264b4d5c334c.png">